docs/getting-started.md

```diff
@@ -74,6 +74,18 @@ An Azure Service Principal is needed for populating the controller manifests. Th
   export AZURE_CLIENT_SECRET=<Password>
   export AZURE_LOCATION="eastus"
   ```
+
+  5. Set the name of the AzureCloud to be used, the default value that would be used by most users is "AzurePublicCloud", other values are:
+
+  - ChinaCloud: "AzureChinaCloud"
```

Comment:
nit: looks like an extra leading space on the list item

Reply:
I had to do that for some reason to make the rendered page look aligned

Reply:
Indeed, it looks right when rendered.